### PR TITLE
Update free programming books it

### PR DESCRIPTION
**Pull request by rozzilla**
I've added a PDF containing a Javascript course.

### CODE REVIEW

**Comment by eshellman** [comment]
seems a bit old, don't you think?

**Comment by rozzilla** [comment]
As I say in this cases: "Old but gold" 😀

**Comment by eshellman** [comment]
What we've tried to do in other cases of "old but gold" is add a note of the year of publication. See #1926 If you can't find a year of publication, use an estimate. Thanks for the extra effort!

**Comment by rozzilla** [comment]
Ok, I've added it ;)

File: README.md [hunk 1]
```diff
@@ -85,3 +85,7 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 + [Free Programming Playgrounds](/free-programming-playgrounds.md)
+
+## Licence
```
Comment (eshellman) [inline README.md]: License for free-programming-books repo is CC BY

**Comment by eshellman** [comment]
Thanks for this. Good idea to note the license in the README, but the British spelling is too confusing, given that github uses American spelling.

**Pull request merged**